static/css/com/layout/header.scss

```diff
@@ -94,29 +104,31 @@
 }
 
 .global-header-panel {
-  @include clearfix;
   box-sizing: border-box;
-  padding: $unit 0;
-  height: 60px;
+  display: flex;
+  width: 100%;
```

Comment:
If `global-header` does not have flex, it won't required, yes?

Reply:
Yes, removed it too)